app/src/main/java/com/group80/uoftinder/RecViewInterface.java

```diff
@@ -0,0 +1,14 @@
+package com.group80.uoftinder;
+
+/**
+ * A public interface that is implemented by RecommendationView.
+ */
+public interface RecViewInterface {
+    // set first displayed user
+    void setDisplayedUser(User displayedUser);
+    // display user
+    void showUser(User displayedUser);
+    // no more compatible users
+    void noCompatibleUser();
+
```

Comment:
Nice, I like the use of this interface as it adheres to the Clean Architecture by allowing the RecommendationPresenter to call the view, through this interface.

Reply:
Yes, will definitely need to wait to get the generate compatibility list before testing.  The unit tests currently are not functional because I am still waiting for other people's code to be merged in main.